### DISCUSSION TITLE
Fumanchu - Handlebars + Helpers Together ❤️

### DISCUSSION THREAD

**Discussion by jaredwray** [handlebars-lang/handlebars.js#2008] [General]
[Handlebars](https://github.com/handlebars-lang/handlebars.js) + [Handlebars-helpers](https://github.com/helpers/handlebars-helpers) (helpers are now maintained in this project) combined into a single package. Easily use in your nodejs or browser projects as it is a drop in replacement when using handlebars directly.

Just `npm i @jaredwray/fumanchu --save`

Then use it in your code like so:
```javascript
var handlebars = require('@jaredwray/fumanchu');
var template = handlebars.compile('{{#if (eq foo "bar")}}<p>Foo is bar</p>{{/if}}');
var html = template({foo: 'bar'});
console.log(html);
```

## Features
* It (handlebars-helpers) are maintained monthly!
* It has 100% code coverage
* It is a drop in replacement for your handlebars code as it contains Handlebars!

Check it out here: https://github.com/jaredwray/fumanchu

**Comment by jaylinski** [comment]
Nice! Can you send a PR to update our handlebars-helpers link? :pray: 

https://github.com/handlebars-lang/handlebars.js/blob/1fc4ef09c1ac0ffb4c0f88ca685f44d1e0f32f89/README.markdown?plain=1#L136

**Reply by jaredwray** [reply]
Done! https://github.com/handlebars-lang/handlebars.js/pull/2009
